src/components/layout/organize/AllCampaignsLayout.tsx

```diff
@@ -15,12 +15,12 @@ const AllCampaignsLayout: FunctionComponent<AllCampaignsLayoutProps> = ({ childr
     return (
         <TabbedLayout
             baseHref={ `/organize/${orgId}/campaigns` }
+            defaultTab="summary"
             fixedHeight={ fixedHeight }
-            subtitle="subtitle for all campaigns tabs" //TODO: generate subtitle
             tabs={ [
-                {  defaultTab: true, href: `/`, label: 'summary' },
-                { href: `/calendar`, label: 'calendar' },
-                { href: `/archive`, label: 'archive' },
+                { href: `/`, label: 'summary', messageId: 'layout.organize.campaigns.summary' },
+                { href: `/calendar`, label: 'calendar', messageId: 'layout.organize.campaigns.calendar' },
+                { href: `/archive`, label: 'archive', messageId: 'layout.organize.campaigns.archive' },
```

Comment:
I'm starting to reconsider this. Here are my objections, and the conclusion:

The `label` property used to be for deciding the (eventually) human-readable label of the tab. Now it's more like a key or ID, but still called label.

The only reason why we need it is so that we can set the `defaultTab`. But I feel like we could use the `href` for that, couldn't we?

In fact, if we use `href` and just set the default value for `defaultTab` to `'/'`, we would very rarely have to change it.

So in conclusion, I think we should remove `label` and use `href` as the "key" instead. Do you think that's a good idea?

Reply:
no you're right, much better this way. all three properties are unique for each tab and only href and messageId are needed